src/plugins/autolink.js

```diff
@@ -24,6 +24,8 @@ if (!CKEDITOR.plugins.get('ae_autolink')) {
 
 	const REGEX_LAST_WORD = /[^\s]+/gim;
 
+	// Seen at
+	// https://stackoverflow.com/questions/5717093/check-if-a-javascript-string-is-a-url/49849482#5717133/
```

Comment:
Pro-Tip™: The "share" link on Stack Overflow answers will give you a shortlink:

https://stackoverflow.com/a/5717133/2103996

Reply:
Prop-Tip accepted thanks!

Reply:
`React.Prop-Tip`